main_app/templates/main_app/common.html

```diff
@@ -21,6 +21,9 @@
       href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"
```

Comment:
The `login` page is getting distorted. Please fix it.
![image](https://user-images.githubusercontent.com/53635792/111617855-ac326880-8809-11eb-8032-314899e5d430.png)